packages/frontend/src/context/positions.tsx

```diff
@@ -108,9 +109,10 @@ const PositionsProvider: React.FC = ({ children }) => {
       : new BigNumber(0)
   }, [firstValidVault, oSqueethBal.toString(), positionType, shortVaults?.length, squeethAmount.toString()])
 
-  const shortDebt = useMemo(() => {
-    return positionType === PositionType.SHORT ? squeethAmount : new BigNumber(0)
-  }, [positionType, squeethAmount.toString()])
+  //meaning if there is minted squeeth sold, need to be subtracted
+  const finalShortSqueeth = useMemo(() => {
+    return mintedSqueeth > mintedDebt ? openShortSqueeth.plus(mintedSqueeth).minus(mintedDebt) : openShortSqueeth
```

Comment:
what is the difference between mintedSqueeth and mintedDebt? Is mintedSqueeth the total amount of all squeeth minted and mintedDebt the amount of minted squeeth that's been sold?